ClosedXML.Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -11,6 +11,7 @@
 namespace ClosedXML.Tests
 {
     [TestFixture]
+    [SetCulture("")]
```

Comment:
Dtto, by setting a culture we are potentially masking an error in implementation when current culture differs from invariant one. Although in this case it is a problem that the test takes a string and tries to convert it to number and that conversion should be done through local culture (not a great idea, but part of the spec).

> ECMA-36718.17.2.6 
> An implementation is permitted to provide an implicit conversion from string-constant to number. However, the
rules by which such conversions take place are implementation-defined. [Example: An implementation might
choose to accept "123"+10 by converting the string "123" to the number 123. Such conversions might be locale-
specific in that a string-constant such as "10,56" might be converted to 10.56 in some locales, but not in others,
depending on the radix point character. end example]

Most of tests use CultureInfo.CurrentCulture, so I guess its better to update TryGetValue_Nullable
```
            ws.Cell("A3").SetValue(2.5.ToString(CultureInfo.CurrentCulture));
```